snuba/settings/__init__.py

```diff
@@ -198,8 +198,13 @@
 
 MAX_ROWS_TO_CHECK_FOR_SIMILARITY = 1000
 
+# Cutoff time from UTC 00:00:00 to stop running optimize jobs to avoid spilling over to the next day.
 OPTIMIZE_JOB_CUTOFF_TIME = timedelta(hours=23)
 
+# Cutoff time from UTC 00:00:00 to stop running optimize jobs in parallel to avoid running in parallel
+# when peak traffic starts.
+PARALLEL_OPTIMIZE_JOB_CUTOFF_TIME = timedelta(hours=11)
```

Comment:
We cannot do this past 9 am UTC. That is the time we start the weekly report queries on monday morning.
Also We cannot do this from the start of the job as, when we start the job (which is around 1 am UTC) the TTL merge is still on going generally (it takes 2.5 hours now). Which means at the start of the job we still have 2 merges already on going. I would have a parameter to decide when to start the parallel section of this job and maybe start around 3 am UTC

Reply:
Added buckets for optimization which define different parallel and cutoff times